_data/layout.yml

```diff
@@ -1 +1 @@
-promotion: study-group_11
+promotion: study-group_12
```

Comment:
oh wait, will this show the correct promotion?

Reply:
Yes it works – taken from the last line of the study-group.md file I guess:

```
---
title: "Study Group #12: applied NLP and non-verbal robots"
location: "World Trade Center Stockholm, Kungsbron 1"
time: "18:00-20:00, January 16"
speakers:
  - per-klaar
  - taras-kucherenko
organizers:
  - anton-osika
---
`study-group_12` content.
```

Reply:
![image](https://user-images.githubusercontent.com/4467025/50679415-88592280-1003-11e9-9446-2b93cba21574.png)